activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/validation/process/BpmnModelServiceTaskImplementationValidator.java

```diff
@@ -56,12 +57,8 @@ public BpmnModelServiceTaskImplementationValidator(ConnectorModelType connectorM
         List<String> availableImplementations = getAvailableImplementations(validationContext);
         //TODO: hardcoded decision table added -> fix this after implementation for decision table will change
         availableImplementations.add("dmn-connector.EXECUTE_TABLE");
-
         availableImplementations.add("script.EXECUTE");
```

Comment:
I am not sure if we should include this in the service task implementation instead of treating them differently. What do you think @mmartinadanx?

Reply:
Moved